packages/performance/docs/events.md

```diff
@@ -0,0 +1,126 @@
+# Events
+
+## Lifecycle Events
+
+### Time to First Byte
+
+Use enum `EventType.TimeToFirstByte`
+
+The time until the server sends the first part of the response.
+[Learn more](https://developers.google.com/web/tools/chrome-devtools/network-performance/understanding-resource-timing)
+
+### First Paint
+
+Use enum `EventType.TimeToFirstPaint`
+
+The time until the browser renders anything that differs from the previous page.
+[Learn more](https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics#first_paint_and_first_contentful_paint)
+
+### First Contentful Paint
+
+Use enum `EventType.TimeToFirstContentfulPaint`
+
+The time until the browser renders the first bit of content from the DOM.
+[Learn more](https://developers.google.com/web/tools/lighthouse/audits/first-contentful-paint)
+
+### DOM Content Loaded
+
+Use enum `EventType.DomContentLoaded`
+
+The time until the DOM has been entirely loaded and parsed.
+[Learn more](https://developer.mozilla.org/en-US/docs/Web/Events/DOMContentLoaded)
+
+### First Input Delay
+
+Use enum `EventType.FirstInputDelay`
+
+The time from when a user first interacts with your site to the time when the browser is able to respond to that interaction.
+[Learn more](https://developers.google.com/web/updates/2018/05/first-input-delay)
+
+### Load Event
+
+Use enum `EventType.Load`
+
+The time until the DOM and all its styles and synchronous scripts have loaded.
+[Learn more](https://developer.mozilla.org/en-US/docs/Web/Events/load)
+
+## Navigation Events
+
+### Long Task
+
+Use enum `EventType.LongTask`
+
+Any task that take 50 milliseconds or more.
+[Learn more](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceLongTaskTiming)
+
+### Script Download Event
+
+Use enum `EventType.ScriptDownload`
+
+The time spent downloading a script resource.
```

Comment:
I think it's probably worth discussing what other fields are here? In the case of script downloads, I think there's also some metadata available.